client/src/components/ElementEditor/Header.js

```diff
@@ -36,19 +63,39 @@ class Header extends Component {
           </div>
           <h3 className="element-editor-header__title">{title}</h3>
         </div>
+        <FormActionComponent
+          onClick={this.handleDelete}
+          title={null}
```

Comment:
This seems like a bug in admin 😒

Reply:
To be fair, we could probably just omit the title prop

Reply:
You can - but you would also expect it to be passed through to the actual button that gets rendered...

Reply:
It already passed through a few attributes: https://github.com/silverstripe/silverstripe-admin/blob/1/client/src/components/FormAction/FormAction.js#L22-L26